packages/circuit-ui/components/Card/Card.tsx

```diff
@@ -54,9 +53,6 @@ const shadowStyles = ({ shadow }: CardProps) => {
 };
 
 const spacingStyles = ({ theme, spacing = 'giga' }: StyleProps & CardProps) => {
-  if (!spacing) {
-    return null;
-  }
```

Comment:
This is here on purpose for the [static style extraction](https://circuit.sumup.com/?path=/story/features-static-css--page). 

I doubt we'll ever finish or use it, so we can add a note to remove it when we rip out this feature completely.

Reply:
Oops, sorry, I didn't realize it. I'll revert this change and add a comment to explain it

Reply:
Done in e9c4e5c